src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -365,11 +365,12 @@ public void TestUserCredentialsPropertiesOnWindows()
             p.StartInfo.LoadUserProfile = true;
             p.StartInfo.UserName = username;
             p.StartInfo.PasswordInClearText = password;
+            bool hasStarted = false;
 
             SafeProcessHandle handle = null;
             try
             {
-                p.Start();
+                hasStarted = p.Start();
```

Comment:
There's no catch block, so the test would still fail... but if such an exception did occur, you'd want to delete the user still rather than leaving the user on the machine.